xwiki-platform-core/xwiki-platform-tour/xwiki-platform-tour-ui/src/main/resources/TourCode/TourJson.xml

```diff
@@ -38,7 +38,7 @@
   <hidden>true</hidden>
   <content>{{include reference="TourCode.Macros"/}}
 
-{{velocity}}
+    {{velocity wiki="false"}}
```

Comment:
Why the indentation? And BTW, you could instead set ``output="false"`` and call the ``#jsonResponse()`` Velocity macro at the end to write the JSON directly to the HTTP response.